.github/workflows/publish.yaml

```
@@ -0,0 +1,27 @@
+name: publish Node.js Package to gpr
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+  release:
+    types: [created]
```

Let’s be consistent with the syntax:

```suggestion
  push:
    tags:
      - 'v*.*.*'
  release:
    types:
      - created
```